posthog/temporal/batch_exports/snowflake_batch_export.py

```diff
@@ -167,10 +167,22 @@ def load_private_key(private_key: str, passphrase: str | None) -> bytes:
         )
     except (ValueError, TypeError) as e:
         msg = "Invalid private key"
+
         if passphrase is not None and "Incorrect password?" in str(e):
             msg = "Could not load private key: incorrect passphrase?"
         elif "Password was not given but private key is encrypted" in str(e):
             msg = "Could not load private key: passphrase was not given but private key is encrypted"
+        elif "Password was given but private key is not encrypted" in str(e):
+            if passphrase == "":
+                try:
+                    p_key = load_private_key(private_key, None)
+                except (ValueError, TypeError):
+                    # Proceed with top level handling
+                    pass
+                else:
+                    return p_key
```

Comment:
Although it should not be possible, this is done this way to catch the niche scenario of somebody actually using empty string as a passphrase.

I don't think this is possible, but `load_pem_private_key` does accept `""` as a passphrase so I just have to assume it _could_ happen.